### PR TITLE
Adding home page

### PR DESCRIPTION
**Pull request by Michal-Shamir**
Changing the existing index.html file to a home page of our website
Adding base.html file to templates- containing Bootstrap import and shared properties to all pages. 
Adding courses and sign up anchor tags- pages to be filled. 
Creating static/img folder containing the logo and background

-should consider changing the tutor logo to a more professional one made with photoshop.
-home page design is primary- would like suggestions and opinions. 

homepage

### CODE REVIEW

**Comment by Michal-Shamir** [comment]
@boaz0 
would be happy if you could review my pr

File: tutor/templates/base.html
Comment (boaz0) [inline tutor/templates/base.html:25]: I am wondering, why did you choose "mark" here? Is it because of its background? If so, there are better ways to do it.

IMHO, discard the "mark" tags and add styles to "a" adding background and padding to make it looks better.

File: tutor/templates/index.html
Comment (boaz0) [inline tutor/templates/index.html:7]: to be more consistent, can you replace the "b" tag to the "strong" tag.

File: tutor/templates/index.html
Comment (boaz0) [inline tutor/templates/index.html:10]: It is OK but I will probably use https://getbootstrap.com/docs/4.5/components/jumbotron/ and add `opacity: 0.8` to its style. 

image

**Comment by Michal-Shamir** [comment]
@boaz0  thank you, applied the changes , hope it's better.

**Review by boaz0 — APPROVED** [review]
LGTM

**Review by ifireball — CHANGES_REQUESTED** [review]
I'm only seeing a template in this PR - and it looks good as far as I can see - but where is the view part of the application and the route to display it? Is there another ticket to handle that?

**Comment by ifireball** [comment]
Good work on a detailed and informational PR description!

**Comment by Michal-Shamir** [comment]
@ifireball the route to display the home page already exists (was created to present the hello world message) , but I changed it's name now. 
i see that in PR #7  routing changes are done so I guess I'll change them again once it is approved

**Review by ifireball — APPROVED** [review]
I see some small issues with files missing a newline at the end of the file - please configure your text editor to add this automatically.

I will not block the PR on this though.

**Pull request merged**
